tensorflow/lite/experimental/acceleration/configuration/configuration.proto

```diff
@@ -622,6 +624,14 @@ message CPUSettings {
   optional int32 num_threads = 1 [default = -1];
 }
 
+// Arm NN Delegate Settings
+// More information about Arm NN delegate options can be found in https://arm-software.github.io/armnn/latest/delegate.xhtml
+message ArmNNSettings {
+  optional string backends = 1;
+  optional bool fastmath = 2;
```

Comment:
The current documentation here does not make the following things clear:
(1) What effect does this `fastmath` field have?
(2) What is the relationship between this field and the `"enable-fast-math"` parameter in `additional_parameters`?
E.g. if this field is set to `true`, but `"enable-fast-math"` is set to `"false"`, which takes precedence?

Reply:
ditto for the 'backends' setting

Reply:
We will update our documentation to make this clear to users of the Opaque Delegate